extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml

```diff
@@ -29,8 +29,8 @@
         <version>1.0</version>
 
         <configuration>
-          <nodeVersion>v5.4.0</nodeVersion>
-          <npmVersion>3.3.12</npmVersion>
+          <nodeVersion>v8.1.2</nodeVersion>
+          <npmVersion>5.0.3</npmVersion>
```

Comment:
Good point: @sjka or @maggu2810 what about this version bump of node and npm? We need it here to provide a stable and customised build of the OpenLayers library. This reduces file size from ~500KB to 230KB.